scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -481,6 +492,49 @@ def post_process_response(self, response: dict):
 
         return result
 
+    def _response_cluster_wait_validation(self, response, result):
```

Comment:
any chance we can add a doc comment with what the method is supposed to do and how it uses the response/result?